lib/application/interfaces/repositories/LogRepository.js

```diff
@@ -21,6 +21,45 @@ const { Repository } = require('..');
 /**
  * LogRepository
  */
-class LogRepository extends Repository {}
+class LogRepository extends Repository {
```

Comment:
@MauritsioRK this is the log interface not the implementation

Reply:
Which is the interface?